Arm/Insts/DPI/Insts.lean

```diff
@@ -6,9 +6,11 @@ Author(s): Shilpi Goel
 import Arm.Insts.DPI.Add_sub_imm
 import Arm.Insts.DPI.PC_rel_addressing
 import Arm.Insts.DPI.Logical_imm
+import Arm.Insts.DPI.Bitfield
 
 /-- List of functions to generate random instructions of the
 DPI class. -/
 def DPI.rand : List (IO (Option (BitVec 32))) :=
   [ DPI.Add_sub_imm_cls.rand,
-    DPI.Logical_imm_cls.rand ]
+    DPI.Logical_imm_cls.rand,
+    DPI.Bitfield_cls.ubfm.rand ]
```

Comment:
@nebeid Replacing this with `Bitfield_cls.rand` should deal with the call-out, I think.